kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -174,9 +174,10 @@ define([
                 } else {
                     paramsBus.emit('parameter-changed', {
                         parameter: parameterSpec.id,
-                        newValue: newValue,
+                        newValue,
+                        newDisplayValue: message.newDisplayValue,
```

Comment:
Most of the changes look like this - making sure the `displayValue` gets percolated through various bus messages.